# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookiePersistentStorage.h
# Commit: 36aa6faea947
# Full Hash: 36aa6faea9477a13b6d94e396596697e56675018
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-19 03:53:26
# Regressor Bug: 1839918
# File Overlap Count: 6
# Description:
#   Bug 1839918 - Refactor "cookie-changed" notifications to use nsICookieNotification. r=extension-reviewers,cookie-reviewers,sessionstore-reviewers,bvandersloot,edgul,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185300
# ==============================================================================

diff -r 905592ffa91d -r 36aa6faea947 netwerk/cookie/CookiePersistentStorage.h
--- a/netwerk/cookie/CookiePersistentStorage.h	Mon Sep 18 11:25:24 2023 +0000
+++ b/netwerk/cookie/CookiePersistentStorage.h	Mon Sep 18 11:55:08 2023 +0000
@@ -72,8 +72,8 @@
  protected:
   const char* NotificationTopic() const override { return "cookie-changed"; }
 
-  void NotifyChangedInternal(nsISupports* aSubject, const char16_t* aData,
-                             bool aOldCOokieIsSession) override;
+  void NotifyChangedInternal(nsICookieNotification* aNotification,
+                             bool aOldCookieIsSession) override;
 
   void RemoveAllInternal() override;
 